fabric-item-groups-v0/src/main/java/net/fabricmc/fabric/impl/itemgroup/FabricCreativeGuiComponents.java

```diff
@@ -47,24 +48,34 @@
 		Type type;
 
 		public ItemGroupButtonWidget(int x, int y, Type type, CreativeGuiExtensions extensions) {
-			super(x, y, 10, 11, type.text, (bw) -> type.clickConsumer.accept(extensions));
+			super(x, y, 11, 11, type.text, (bw) -> type.clickConsumer.accept(extensions));
 			this.extensions = extensions;
 			this.type = type;
 			this.gui = (CreativeInventoryScreen) extensions;
 		}
 
 		@Override
 		public void render(int mouseX, int mouseY, float float_1) {
+			// Copied From Vanilla
+			this.isHovered = mouseX >= this.x && mouseY >= this.y && mouseX < this.x + this.width && mouseY < this.y + this.height;
 			this.visible = extensions.fabric_isButtonVisible(type);
 			this.active = extensions.fabric_isButtonEnabled(type);
 
 			if (this.visible) {
 				MinecraftClient minecraftClient = MinecraftClient.getInstance();
+				TextRenderer textRenderer = minecraftClient.textRenderer;
 				minecraftClient.getTextureManager().bindTexture(BUTTON_TEX);
 				GlStateManager.disableLighting();
 				GlStateManager.color4f(1F, 1F, 1F, 1F);
-				this.blit(this.x, this.y, (type == Type.NEXT ? 12 : 0), (active ? 0 : 12), 12, 12);
-
+				this.blit(this.x, this.y, 0, (active ? (isHovered() ? 24 : 0) : 12), 11, 11);
+				int textColor = 14737632;
```

Comment:
Could these be converted to hex?

Reply:
This is copied from vanilla but this is doable through manual conversion

Reply:
![image](https://user-images.githubusercontent.com/6166773/62173349-972e5c00-b2ea-11e9-9b73-9a7ea39f34a3.png)

Reply:
why did i not know